docs/categories/index.html

```diff
@@ -30,9 +30,13 @@
 </script>
 <script async src='//www.google-analytics.com/analytics.js'></script>
 
+   <script src="./SearchIndex.js"></script>
+   <script src="./js/search.js"></script>
```

Comment:
Place JavaScript at the bottom of the page to prevent it from blocking rendering. Google Analytics is a special case because it's loading async, but your code needs the DOM and won't be able to load async. See [Remove render-blocking JavaScript](https://developers.google.com/speed/docs/insights/BlockingJS).